mars/deploy/oscar/file-logging.conf

```diff
@@ -66,7 +66,7 @@ level=DEBUG
 args=(sys.stderr,)
 
 [handler_file_handler]
-class=mars.deploy.oscar.file_logging_handler.FileLoggingHandler
+class=FileHandler
```

Comment:
Would it be better to use RotatingFileHandler ?

Reply:
One logging file is easier for us to process, like getting head or tail content and downloading all logs.